next/pages/index.tsx

```diff
@@ -1,71 +1,8 @@
 import type { NextPage } from 'next'
```

Comment:
Hi @BruceRodrigues it looks like prettier isn't configured on your IDE - can you please check and resolve `next/pages/index.tsx` and other files for formatting.